website/blog/2020/12/11/beike-how-to-build-gateway-based-on-apache-apisix.md

```diff
@@ -7,7 +7,7 @@ keywords:
 - Apache APISIX
 - Kong
 - Nginx
-description: Beike uses Apache APISIX as the API gateway of the production system, processing over 100 million production traffic every day, with excellent performance and stability. It happens that Apache APISIX has just joined the Apache incubator. In addition to congratulations, I would like to share why I chose Apache APISIX in the first place and some experiences in the process of using it.
+description: This article describes why Beike chose Apache APISIX as the API gateway in the first place, and some experiences in the process of using it.
```

Comment:
Hi 👋 May I know why we should follow this style like `This article describes xxx`? This is because this field will be rendered in the HTML Head tag and used by the Search Engine. 

Even though this version is shorter, I can't see "real" improvements for SEO or other sides. 🤔

Reply:
I will revise the `description` shortly.

Reply:
The newer ones contains keywords like "API Gateway", and make description more abstract. I would say yes to these changes.

> I can't see "real" improvements for SEO or other sides.

You can't really see them until a couple months after modification, I think this is a nice try.